### ISSUE TITLE
Maven central release

### ISSUE THREAD

**Issue by dragetd** [goekay/CompositeJKS#1]
These are some useful additions! Maybe we can try pushing it also into maven central, then it is easier to use in other projects.

**Comment by goekay** [comment]
agreed. i will look into it. and for the time being, you might check out [jitpack.io](https://jitpack.io/docs/) to use/import dependencies that are github projects into your project. jitpack pulls the repo, builds it and hosts its at their own maven repo.

**Comment by dragetd** [comment]
Uh, thanks for the hint. Fun project. Gives a bit of that golang feeling of dependency-management… xD (this is not necessarily a good thing… and I have been doing go the past year. :P)
